src/css/style.css

```diff
@@ -3,3 +3,71 @@
   margin: 0;
   padding: 0;
 }
+
+body {
+  background-color: #faf7fc;
+}
+
+h1 {
+  font-size: 42px;
+  /* font-family:"Times New Roman", serif; */
+  font-family: 'Onest', sans-serif;
+  font-weight: 600;
+  color: #202020;
+  margin-left: 120px;
+  margin-bottom: 100px;
+}

```

Comment:
use a single margin property  example margin: 10px 10px; specifies margin top and bottom as 10px and right and left as 10px,